### ISSUE TITLE
@ mentions don't work on desktop

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#82]
steps: 

1. @ mention someone

expected: 

1. user selector appears
2. name is highlighted for sender 
3. name is highlighted for recipients
4. notification appears

actual: nothing happens

**Comment by holmesworcester** [comment]
@pdurbin what do you think about these mockups? 

[Loom](https://www.loom.com/share/d98ee90d9c5a49b48542388201e81757) and [Figma](https://www.figma.com/file/nDBc7Qo2dZUTIV2vHMupxi/Mentions-for-notifying-someone?type=design&node-id=13%3A37&mode=design&t=55cuRCAiHIlnoHUi-1)

**Comment by pdurbin** [comment]
Looks good to me!

**Comment by ethanc8** [comment]
I think we should allow users to disable notifications in DMs except when the other person @-mentions them in the DM, if we ever support DMs with more than one person. This would allow people to create small, single-channel groupchats easily without having the bad downside of always notifying someone.

**Comment by holmesworcester** [comment]
@ethanc8 yes! We'll definitely distinguish between 1-on-1 DMs and group DMs, for notification purposes.

**Comment by ethanc8** [comment]
I am working on trying to implement this.

**Comment by ethanc8** [comment]
https://github.com/ethanc8/quiet/tree/feat/82-at-mentions

**Comment by ethanc8** [comment]
My questions -
* What is the preferred way to get a list of all the users in the channel? Currently I am getting the list of all users using `const allUsers: Record<string, User> = useSelector(users.selectors.allUsers)`.
* Which identifier uniquely identifies a user in a way that is invariant across devices and username changes?
* How do I get a user's avatar? I see `waitForAvatar`, but this is for a specific message. Also, is there a fallback avatar I am supposed to use?

**Comment by holmesworcester** [comment]
@adrastaea has been working on a big refactor of how we do users. taea, can you answer this?

**Comment by holmesworcester** [comment]
@ethanc8 you should use the `jdenticon` placeholder avatar as a fallback avatar if an avatar is not available.

**Comment by ethanc8** [comment]
> [@ethanc8](https://github.com/ethanc8) you should use the `jdenticon` placeholder avatar as a fallback avatar if an avatar is not available.

How do I get the avatar if it *is* available? Do I need to search the message history for their latest message?
